manager/gui/tests/unit/LrSystemHealthConfiguration.spec.ts

```diff
@@ -0,0 +1,181 @@
+import { mount, createLocalVue } from "@vue/test-utils";
+import LrSystemHealthConfiguration from "@/components/configuration/LrSystemHealthConfiguration.vue";
+import Vuetify from "vuetify";
+import { Vue } from "vue-property-decorator";
+
+describe("LrSystemHealthConfiguration", () => {
+  let wrapper: any;
+  const localVue = createLocalVue();
+  let vuetify: Vuetify;
+
+  beforeAll(() => {
+    vuetify = new Vuetify();
+    wrapper = mount(LrSystemHealthConfiguration, { localVue, vuetify });
+  });
+
+  test(`Title should be "System Health"`, () => {
```

Comment:
This test might not be required and also this would fail if internationalization in implemented.

Reply:
This test has been removed.